atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -560,6 +561,17 @@ def get_build(self, metadata):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if help_result is not None:
+            try:
+                cgit_url = self.workflow.source.uri.replace('/rpms/', '/cgit/rpms/')
```

Comment:
A template string should allow users to use whatever hosting solution they'd like. We could format it with what we know: `commit`, `help_file_name`, others?